index.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
```

Comment:
Please re-run `apict copyright` to fix the copyright headers (there should only be a single year for this file, for example).